### PR TITLE
Generalize analysis

### PR DESCRIPTION
**Pull request by jgallowa07**
Running for me for any model. Also structures the output intermediate files into subdirectories by replicate seed.

### CODE REVIEW

**Comment by jgallowa07** [comment]
@andrewkern this should be good to go if you want to test it out. I'm going to squash the commits now.

**Comment by jgallowa07** [comment]
Closes #21

**Comment by jgallowa07** [comment]
@andrewkern np.

**Comment by jgallowa07** [comment]
@andrewkern Let me know if you want a little more explanation. I also added a `png` of the dag for a simple run for clarification.

File: n_t/msmc.py [hunk 1]
```diff
@@ -30,10 +30,12 @@ def write_msmc_file(path, num_sampled_genomes_msmc):
     """
     for sample_size in num_sampled_genomes_msmc:
         ts = prune_tree_sequence(path, sample_size)
-        sep = path.split(".")
-        chrom = sep[1]
-        sep.insert(1,str(sample_size))
-        output = ".".join(sep) + ".multihep.txt"
+        dirr = os.path.dirname(path)
```
Comment (jeromekelleher) [inline n_t/msmc.py]: Can do this more cleanly using pathlib.

File: n_t/simulations.py [hunk 2]
```diff
@@ -6,17 +6,14 @@
 from stdpopsim import homo_sapiens
 
 
-def homo_sapiens_Gutenkunst(path, seed, chrmStr, sample_size=20):
-    chrom = homo_sapiens.genome.chromosomes[chrmStr]
-    model = homo_sapiens.GutenkunstThreePopOutOfAfrica()
-    # model.debug()
-
-    # Currently sampling 20 individuals from a single popn.
+def simulate(out_path, species, model, genetic_map, seed, chrmStr, sample_size=20):
```
Comment (jeromekelleher) [inline n_t/simulations.py]: We'll probably be replacing this with calls to the CLI in stdpopsim, so I don't think we need to worry about generalising too much here.

Comment (jgallowa07) [reply n_t/simulations.py]: Agreed, should we just keep it how it is for now?

Comment (jeromekelleher) [reply n_t/simulations.py]: Yep, that's a plan.

File: n_t/msmc.py (hunk 1 above)
Comment (andrewkern) [reply n_t/msmc.py]: @jeromekelleher should I merge this and we can change later?

Comment (jeromekelleher) [reply n_t/msmc.py]: can change later, totally not a blocker.

**Pull request merged**